miner/build.rs

```diff
@@ -0,0 +1,11 @@
+extern crate cc;

```

Comment:
we are using rust 2018 edition, extern crate is no longer needed
```suggestion
use cc::Build;
```